apps/calendar/js/backend/calendar_service.js

```diff
@@ -1,88 +1,104 @@
+define(function(require, exports, module) {
 'use strict';
-importScripts('../ext/alameda.js');
 
-require.config({
-  baseUrl: '/js/backend',
-  paths: {
-    ext: '/js/ext'
-  }
-});
-
-require(['ext/threads'], threads => {
-  var service = threads.service('calendar');
-
-  service.method('start', () => {
-    // TODO
-    console.log('start');
+var Db = require('db');
+var co = require('ext/co');
+var threads = require('ext/threads');
+
+var db = new Db('b2g-calendar');
+var loadDb = db.load();
```

Comment:
On the one hand that makes sense to me but on the other hand I would like to get the process started as soon as possible...